docs/prepping-local-system-runbook-windows.md

```diff
@@ -390,3 +393,5 @@ Status: Downloaded pega-docker.downloads.pega.com/platform/installer:<version>
    `$ docker push <Registry host name:Port>/my-pega-installer:8.4.0`
 
 After you push these three downloaded images to your private Docker registry, you are ready to begin deploying Pega Platform to a support Kubernetes environment. Use the runbook in this Github directory for your deployment.
+
+11. You can follow the steps similar to above when configuring `backingservices` with your deployment from steps 5 through 7 and pull `platform-services/search-n-reporting-service:<version>` instead, to configure Search and reporting service feature as an alternate search solution to `platform/search`.
```

Comment:
Since the naming of this image is long and complex, can you please show the three commands for this image in both Windows and Linux so it is absolutely clear what the required docker formatting is for the backing service image and the return value? For instance, show a good example of what they can retag it to...I am guessing here. But don't just say repeat steps for this one when it's the most complex name. Something like this:

To download your preferred version of the search-n-reporting-service image to your local system, specify the version tag when you enter:
$ docker pull pega-docker.downloads.pega.com/platform/platform-services/search-n-reporting-service:<version>
Digest: <encryption verification>
Status: Downloaded pega-docker.downloads.pega.com/platform/pega:<version>

To retag the search-n-reporting-service image for your deployment with a customized tag that includes your Docker registry host name and a name that is useful to your organization, such as <Registry host name:Port>/my-pega:<Pega Platform version>.

$ docker tag pega-docker.downloads.pega.com/platform/platform-services/search-n-reporting-service:<version>:8.6.0 <Registry host name:Port>/my-pega-search-n-reporting-service:8.6.0

To push the retagged my-pegasearch-n-reporting-service image to your registry, enter:

$ docker push <Registry host name:Port>/my-pegasearch-n-reporting-service:8.4.0